scripts/install_third_party_libs.py

```diff
@@ -87,13 +87,12 @@
 BUF_DIR = os.path.join(
     common.OPPIA_TOOLS_DIR, 'buf-%s' % common.BUF_VERSION)
 PROTOC_DIR = os.path.join(BUF_DIR, 'protoc')
-# Path of files which needs to be compiled by protobuf.
-# Paired with the frontend requirement.
+# List of a tuple represents the path of files that needs to be compiled by
```

Comment:
A list of 2-tuples, each representing a proto file. For each tuple, the first element represents whether buf needs to generate its JS/TS file, and the second element represents the path to the file.

(Also -- probably makes more sense to represent this as a dict from filepath to boolean, and call it PROTO_FILE_COMPILATION_REQS?)

Reply:
Updated to dict, it looks clean though.